components/team_card/team_card_storage.lua

```diff
@@ -48,16 +42,16 @@ function TeamCardStorage.saveToLpdb(args, teamObject, players, playerPrize)
 	lpdbData = Custom.adjustLpdb and Custom.adjustLpdb(lpdbData, team, args, lpdbPrefix) or lpdbData
 
 	-- Store aliases (page names) for opponenets for setting page vars
-	if Table.isNotEmpty(teamObject.aliases) then
+	if Standard.Table.isNotEmpty(teamObject.aliases) then
```

Comment:
I'd prefer to not allow such usages if there are more than one of the same module tho. We already struggle often with linelength and this wouldn't make it any easier